src/multihash.rs

```diff
@@ -49,7 +69,9 @@ pub trait MultihashDigest<const S: usize>:
     /// let hash = Code::Sha3_256.wrap(&hasher.finalize());
     /// println!("{:02x?}", hash);
     /// ```
-    fn wrap(&self, digest: &[u8]) -> Multihash<S>;
+    fn wrap(&self, digest: &[u8]) -> Multihash<S> {
+        Multihash::wrap((*self).into(), digest).unwrap()
```

Comment:
Same here, please add a comment or `expect()` to make clear why it is OK to use `unwrap()` here.